docs/import-data/overview.mdx

```diff
@@ -147,46 +147,13 @@ export default function ConnectToMemgraph() {
         </section>
         <section>
           <h2 id="kafka">
-            How to import streaming data from Kafka into Memgraph?
+            How to import streaming data into Memgraph?
           </h2>
           <p>
             Memgraph comes with full streaming support, and you can connect
-            Memgraph directly to a Kafka stream. Check out{" "}
-            <a href={useBaseUrl("/memgraph/import-data/kafka")}>this guide</a>{" "}
-            to find out how to connect to a Kafka stream and take a look at the
-            examples with different serialization mechanisms:
+            directly to a Kafka, Redpanda or Pulsar stream {" "}<a href={useBaseUrl("/import-data/data-streams/manage-streams.md")}>using Cypher queries</a>{" "} 
+            or {" "}<a href={useBaseUrl("/import-data/data-streams/manage-streams-lab.md")}>using Cypher queries</a>{" "} 
```

Comment:
```suggestion
            or {" "}<a href={useBaseUrl("/import-data/data-streams/manage-streams-lab.md")}>using Memgraph LAB</a>{" "} 
```

Reply:
you need to show me how you did this so I can do it to others :D

Reply:
@vpavicic, when you choose the line(s) to comment in the PR, you can find the first button on the right looking like a piece of paper with + and - sign on it. It is for suggestions. When you rewrite the code, it shows as a suggested change. In the preview tab, you can check whether you did it correctly.

Reply:
or by using:
backtick backtick backtick suggestion
_write your suggested change_
backtick backtick backtick

Reply:
I don't know how to resolve this error with the links..... i don't know how to write the paths